packages/cdk-ecs-alb/lib/index.ts

```diff
@@ -19,7 +19,9 @@ export interface CdkEcsAlbProps {
     tag?: string;
     certificateArn: string;
     serviceDnsRecord?: string;
-    hostedZoneDomainName?: string
+    hostedZoneDomainName?: string;
+    taskCapacity: number[];
```

Comment:
The type of this being an array of numbers I think makes this hard for someone making a frank template to see what index corresponds to what value without looking at the source code. I think it be more usable if you made this it's own type with min, max, and desired clearly defined.